ql/src/java/org/apache/hadoop/hive/ql/session/SessionState.java

```diff
@@ -42,6 +41,10 @@
 import java.util.concurrent.CancellationException;
 import java.util.concurrent.locks.ReentrantLock;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
DO NOT REORDER IMPORTS!

Reply:
how do i revert it to the original?

Reply:
You can just copy this line to its original place...